Examples/Io/Root/include/ActsExamples/Io/Root/RootSeedVertexPerformanceWriter.hpp

```diff
@@ -0,0 +1,110 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Vertexing/Vertex.hpp"
+#include "ActsExamples/EventData/SimParticle.hpp"
+#include "ActsExamples/Framework/WriterT.hpp"
+
+#include <mutex>
+#include <vector>
+
+class TFile;
+class TTree;
+
+namespace ActsExamples {
+
+/// @class RootSeedVertexPerformanceWriter
+///
+/// Writes out the number of reconstructed seed vertices along with
+/// the number of truth vertices in detector acceptance.
+/// Additionally it matches the reco vertices to their truth vertices
+/// and write out the difference in x, y, and z position.
+class RootSeedVertexPerformanceWriter final
```

Comment:
any chance to use the existing vertex performance writer?

Reply:
Hi @andiwand and @asalzburger, I've change it to `VertexPerformanceWriter`. I needed to modify `VertexPerformanceWriter` a little bit so it can handle vertices without any reconstructed tracks.
(Default behaviour of `VertexPerformanceWriter` is unchanged.)